pkg/controller/zookeepercluster/zookeepercluster_controller.go

```diff
@@ -210,14 +214,58 @@ func (r *ReconcileZookeeperCluster) reconcileStatefulSet(instance *zookeeperv1be
 		}
 		return r.updateStatefulSet(instance, foundSts, sts)
 	}
-	return nil
```

Comment:
Why have we removed this return statement?

Reply:
This statement is unreachable.